ports/libsimpleservo/api/src/lib.rs

```diff
@@ -185,12 +185,11 @@ pub struct ServoGlue {
 }
 
 pub fn servo_version() -> String {
-    let cargo_version = env!("CARGO_PKG_VERSION");
-    let git_info = option_env!("GIT_INFO");
-    match git_info {
-        Some(info) => format!("Servo {}{}", cargo_version, info),
-        None => format!("Servo {}", cargo_version),
```

Comment:
Was this GIT_INFO optional to support building servo from source tarballs? Are we removing support for that with this patch?

Reply:
This is a good. I'm cannot think of any time we build from source tarballs, but I've updated the code to handle this case explicitly. Now the version is `nogit`.